src/components/DarDetails.js

```diff
@@ -0,0 +1,48 @@
+import { PureComponent } from 'react';
+import { div, hh, span } from 'react-hyperscript-helpers';
+
+const styles = {
+    consentDataAlt: {
+        'font-size': 19,
+        'font-style': 'normal',
+        'margin': '10px 0 30px 56px',
+        'color': '#333333',
+        'font-weight': 'normal',
+        'line-height': '27px'
```

Comment:
All of these dashed style properties should be camel cased: 

<img width="1424" alt="Screen Shot 2019-07-11 at 1 39 33 PM" src="https://user-images.githubusercontent.com/116679/61072454-545f1f80-a3e1-11e9-9835-14c75d3a0ffb.png">